utils/navigator_msgs/srv/MessageHeartbeat.srv

```diff
@@ -0,0 +1,2 @@
+---
+string message
```

Comment:
Is this used? Heartbeat should be sent automatically at fixed rate, not triggered externally with a service